commands/load.js

```diff
@@ -2,20 +2,30 @@ const { communications, force, finance, specialInterest } = require('../data/car
 const { underline } = require('cli-color');
 const formatOutput = require('../util/format');
 const error = require('../util/error');
+const { SEED_LENGTH, RANGE_MINIMUM, RANGE_MAXIMUM, ERROR } = require('../util/validation');
 
 module.exports = (seed) => {
```

Comment:
we can modify the params of this function to take a second param that's a boolean if we don't want to validate when the seed is passed via `generate.js`